File: src/main/java/tw/idv/fy/utils/intentchooser/example/ExampleActivity.java
package tw.idv.fy.utils.intentchooser.example;

import android.app.Activity;
import android.os.Bundle;

import tw.idv.fy.utils.intentchooser.IntentChooserActivity;

public class ExampleActivity extends Activity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        IntentChooserActivity.OpenDefault(this, getString(R.string.MDN_pdf));
        finish();
    }
}

File: src/main/java/tw/idv/fy/utils/intentchooser/example/ExampleActivity.kt
package tw.idv.fy.utils.intentchooser.example

import android.app.Activity
import android.content.Intent
import android.os.Bundle
import tw.idv.fy.utils.intentchooser.IntentChooserActivity

class ExampleActivity : Activity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        startActivity(Intent(this, IntentChooserActivity::class.java))
    }
}
